library/.env.example

```
@@ -1,4 +1,3 @@
 JS_HOST=js-dev.basistheory.com
-API_HOST_PROD=api.basistheory.com
-API_HOST_DEV=api-dev.basistheory.com
-API_HOST_LOCAL=localhost:3333
+API_HOST=localhost:3333
+ELEMENTS_HOST=elements-dev.basistheory.com
```

```suggestion
ELEMENTS_HOST=localhost:3000
```
What is your take on this?